core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,6 +25,99 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak" ng-controller="CollectionEditorNavbarBreadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span ng-if="collection">
+        <[collection.getTitle()]>
+        <span class="oppia-navbar-breadcrumb-separator" style="margin-left: 5px;"></span>
+        <span>
+          <[getCurrentTabName()]>
+        </span>
+      </span>
+      <span ng-if="!collection">
+        Loading...
```

Comment:
This is probably fine. I think this is the same behavior as the exploration editor. Since the 'loading' bit is part of the breadcrumb, that doesn't seem too bad to me.